api/swagger.yaml

```diff
@@ -1106,6 +1106,8 @@ definitions:
         type: "boolean"
       Attachable:
         type: "boolean"
+      Ingress:
+        type: "boolean"
```

Comment:
Also I think the current API version is now 1.29, so we need a new section in that document 😅